### ISSUE TITLE
Update /city-resources page design

### ISSUE THREAD

**Issue by smarziano** [smartchicago/chicago-early-learning#810]
Here is an updated version of the City Resources page as part of our redesign work:
![resources](https://user-images.githubusercontent.com/5550969/27447205-c68804f4-5745-11e7-869e-f03da1feee53.png)

My notes about this page:
- I think the title "Find resources" is a little misleading because there isn't an ability to search for and find resources.
- We need to incorporate the logos onto this page
- I am not sure about this format to show content.
- Let's add a "blog" spot on this page. How I envision this being used is to share about things happening at these locations; things that are important for parents with children ages 0-5 but not about early learning. Here are a couple of examples: 
Chicago Department of Public Health might do a campaign for parents to understand how to check for ticks when their kids play outside in the Summer months, or Chicago Public Library might have an event coming up, or Chicago Park District might have a new program specifically for kids under 5 years old.

**Comment by amynicole** [comment]
![resources](https://user-images.githubusercontent.com/7504439/27545547-b235af20-5a55-11e7-827d-ac600a90432a.png)
Incorporated logos and shifted the layout to match the About page for consistency. Blog spot was added as well.

**Comment by smarziano** [comment]
This looks great! Re: the blog spot, let's show that there can be more than one blog. Either by using the same styling that is on the "Latest News" section on the homepage design, or the "Kudos" section on the early learning programs page design. I'm open to other options just as long as it is clear that there are 2-3 blog posts. 

Once that is updated this will be ready to share with our partners for their feedback.

**Comment by smarziano** [comment]
@amynicole & @jenmakes I will be sending this design to our partners for their review. We will need to add Medicaid & SNAP information (see issue #814) to this page. 

I wonder if we should separate these resources into groupings like organizations (Chicago Parks, Chicago Public Library, Chicago Children's Museum, CDPH) vs. programs (Connect4Tots, Medicaid & SNAP, CCOL). That might be unnecessary, though. I just wanted to bring it up for consideration now before I forget, but let's wait to make any changes to page structure until we get partner feedback.

**Comment by jenmakes** [comment]
@smarziano I already went ahead and updated the blog posts on the bottom to match the index page look because we felt it was more clicky and interesting to look at. I did incorporate a carousel as discussed. I'll wait until feedback to incorporate anything new but did want to share what was incorporated so far. https://cl.ly/3R012C012316

**Comment by smarziano** [comment]
Today, our partners and I spoke about restructuring this page because they were less interested in listing organizations and more interested in building this page to be about the types of activities. 

We came up with three categories:
1) **Play** – this would include Chicago Park District & Chicago Children's Museum
2) **Learn** – this would include Chicago Public Library and CCOL
3) **Grow** – this would include Chicago Department of Public Health, Connect4Tots, Medicaid & SNAP (not currently on the page; see issue #814)

They believe all current and future activities and resources could fall into those categories. The blog section at the bottom can stay as it is. Other wishlist items for this page:
- Instead of calling it "City Resources" our partners thought "Around the City" would be fun. We want to stay away from calling these things programs (b/c they are not on Chicago Early Learning search) and I've been thinking of them as "activities"
- They wanted to see the little girl illustration utilized in different categories (the little girl playing, little girl learning, little girl growing... or three different kids participating in the activities but with the same theme of kids in career clothes?), but if that is hard, some way of incorporating illustrations into the 3 categories would be nice to distinguish them.

**Comment by jenmakes** [comment]
@smarziano Something like 'Around the City' is nice. Under those categories, would you still highlight the different organizations evolved? I like the idea of incorporating illustration to lead a section.

**Comment by smarziano** [comment]
Yes, those organizations should be in the related category. Thanks!

**Comment by jenmakes** [comment]
Hi @smarziano I did not get to new illustrations yet but here is the new layout centered around 'Learn,Play,Grow' https://cl.ly/001U3C0g3Z1d

**Comment by smarziano** [comment]
@jenmakes This page is great! Our partners are loving the direction. However, they are not sure if they should organize these sections by organization or if they should add the organizations' programs under each section. They are discussing internally and will get back to us early next week about next steps. Due to this I am adding the "Requires info/approval" tag and assigning it to myself to make sure I check-in.

Regarding the illustrations, our partners did not have any new ideas outside of showing diversity in the kids we choose and incorporating the theme of little kid in career clothes learning, playing, growing. I think even as we wait on the decision whether to add organization vs. program, these illustrations can still be created.

**Comment by smarziano** [comment]
It appears that we are going to continue to keep the page outline of organizations versus changing it to specific programs. Therefore, outside of the illustrations, this page is ready to go.

I am going to keep this issue open to discuss and review illustrations. @jenmakes what is our progress on those?

I am also going to open a new issue to begin implementation of this page design.

**Comment by jenmakes** [comment]
@smarziano here are new illustrations for the City Resources page. My goal was to create fun, diverse children and parent pairs for each section.

learn https://cl.ly/1H350b0e1I35
grow https://cl.ly/1D2N3v123d0j
play https://cl.ly/1w2G40220c08
'Look what WE can do' graphic - not on this page but it also needed an update. I was thinking we can pull all the children here. https://cl.ly/423K3n133K2v
I also tweaked/cleaned-up the index pair: https://cl.ly/3t28142U1J3k

Full page resources https://cl.ly/3l3d13193V3l

Please let me know if you need any tweaks
Jen

**Comment by smarziano** [comment]
These are lovely. The one thing I need to mention is that our partners are expecting to see the children in the career clothes that matches the rest of the marketing campaign. Do you think that is possible in our current time frame? 

I also think it could be unclear or distracting to make all the children in their career clothes, so I am open. I would love to hear your thoughts on this.

**Comment by smarziano** [comment]
After thinking it over and our discussion, I do not think the illustrative kids need to be in career clothes. I, however, would like to see more elements that connect the illustration to the section. I think the first one with the kite is the best because the kite ties it to the "play."

For the "Learn" one maybe it would be nice to add a graduation cap, books, a chalkboard in the background, etc.

For the "Grow" one, it's a little harder. Maybe a ruler?

**Comment by smarziano** [comment]
With the updates to the learn and grow illustrations created and approved, I am going to close this issue. Additional work will happen at the implementation phase (#832)
